src/components/Navigation.tsx

```diff
@@ -1,29 +1,36 @@
+"use client";
+
+import { usePathname } from "next/navigation";
+
 import { NavLink } from "@/components/button";
-import { Icon } from "@/components/icon/Icon";
+import { Icon, IconType } from "@/components/icon/Icon";
 
-const labels = [
-  { href: "/search", label: "話題を検索", icon: <Icon type="Search" /> },
-  { href: "/fav", label: "お気に入り", icon: <Icon type="Heart" /> },
-  { href: "/list", label: "買い物リスト", icon: <Icon type="ShoppingCart" /> },
+const LABELS = [
+  { href: "/search", label: "検索", icon: "Search" },
+  { href: "/fav", label: "お気に入り", icon: "Heart" },
+  { href: "/list", label: "買い物リスト", icon: "ShoppingCart" },
 ];
```

Comment:
as const つければIconコンポに渡すときに型アサーションしなくてもOKになるっす
```suggestion
const LABELS = [
  { href: "/search", label: "検索", icon: "Search" },
  { href: "/fav", label: "お気に入り", icon: "Heart" },
  { href: "/list", label: "買い物リスト", icon: "ShoppingCart" },
] as const;
```

Reply:
初知りでした！めちゃくちゃ便利ですね！このやり方